src/runtime/msg/event.rs

```diff
@@ -124,6 +124,10 @@ pub enum Event {
     LibraryItemNotificationsToggled {
         id: LibraryItemId,
     },
+    LibraryItemMarkedAsWatched {
```

Comment:
please add docs for this item too that it marks it and override the current watched state.